README.md

```diff
@@ -11,6 +11,9 @@ Library that provides common network functionality to GDL components.
 ## Run tests
     sbt test
 
+## Publish local for use in other components
```

Comment:
Hmm...kanskje litt misvisende kommentar, man gjør jo en publishLocal, kun for å teste dette på en komponent før den benytter en offisiell release.

Reply:
Sant nok, men eg trengte å vite det for å teste komponenten lokalt.